pkg/bootstrap/export_test.go

```diff
@@ -0,0 +1,7 @@
+package bootstrap
+
+import "github.com/weaveworks/pctl/pkg/runner"
+
+func SetRunner(r2 runner.Runner) {
```

Comment:
Grumble grumble, I'm not sure I like this. Even package local values should generally be avoided. But I don't have a strong opinion on it.

Reply:
:shrug: